controllers/operatorpolicy_controller.go

```diff
@@ -158,59 +159,110 @@ func (r *OperatorPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	// handle the policy
 	OpLog.Info("Reconciling OperatorPolicy")
 
-	desiredSub, desiredOG, err := r.buildResources(ctx, policy)
+	errs := make([]error, 0)
+
+	conditionsToEmit, conditionChanged, err := r.handleResources(ctx, policy)
+	if err != nil {
+		errs = append(errs, err)
+	}
+
+	if conditionChanged {
+		conditionsToEmit = append(conditionsToEmit, calculateComplianceCondition(policy))
```

Comment:
Could you please add a comment indicating that this is the "final" compliance event to emit of the current policy state? It wasn't immediately obvious to me and that explains why `conditionsToEmit` is named as it is rather than `earlyComplianceEvents`.